.github/workflows/2-deploy-to-staging.yml

```diff
@@ -10,8 +10,7 @@ jobs:
     environment:
       name: deploy/staging
     steps:
-      - name: Send Slack message that deploy is starting
```

Comment:
QQ: What's the rationale for deleting the naming? Seems helpful if a new eng needs to pick up any future workflow work